### ISSUE TITLE
Documentation improvment -- Full Continuous Integration Support

### ISSUE THREAD

**Issue by jirikrepl** [xolvio/chimp#316]
Hi,
I am struggling with setting chimp on Codeship. I would be great if docs say how to set put it.

http://stackoverflow.com/questions/36198667/run-meteor-tests-with-chimp-at-codeship-ci

Thanks a lot

**Comment by ramijarrar** [comment]
I don't think this is related to CI support. You can set the ```screenshotsOnError``` option to get images of failed tests. Have a read of [this](https://chimp.readme.io/docs/command-line-options) article.

**Comment by ramijarrar** [comment]
@jirikrepl Does that answer your question?

**Comment by jirikrepl** [comment]
@ramijarrar Thank you for your answer.

It seems to me that chimp should make screenshots even in its default [configuration](https://github.com/xolvio/chimp/blob/master/src/bin/default.js#L24).
Screenshots should be saved into `.screenshots` directory. I assume it is same directory I run chimp from.

So this command should be sufficient. Result is no screenshots.

    chimp --ddp=http://localhost:3000 --mocha --browser=firefox --path=tests

Also this with same result (no screenshots)

    chimp --ddp=http://localhost:3000 --mocha --browser=firefox --path=tests --screenshotsOnError

Or this, same result (no screenshots)

    chimp --ddp=http://localhost:3000 --mocha --browser=firefox --path=tests --screenshotsOnError=true

**Comment by Sanjo** [comment]
Have your already upgraded Chimp to 0.33.1?

**Comment by jirikrepl** [comment]
Still no luck, I had version 0.31.1, I updated it to 0.33.1.  Would you be so kind and screenshotsOnError functionality on some sample mocha chimp code? 

I mean create some example repository with failing tests that will result in creation of sreenshots. And give me exact `chimp` command

I would be very grateful.

**Comment by thebarty** [comment]
Hi guys,

PLUS ONE for docs on how to save screenshots in any client-modes.

Simply using a ``browser.screenshot()`` does not do anything for me. 
It also would be interesting to know how this would work when using phantomjs.

Any ideas?

**Comment by javascriptlove** [comment]
Please make proper documentation on all command line arguments (like --screenshotsOnError, --screenshotsPath etc.). There's nothing here
https://chimp.readme.io/docs/command-line-options
nor in the link where it says "Cucumber.js CLI documentation" https://github.com/cucumber/cucumber-js#cli

Putting a link to https://github.com/xolvio/chimp/blob/master/src/bin/default.js isn't very nice
